backend/open_webui/models/artifacts.py

```
@@ -207,6 +207,24 @@ def delete_old_processing_records(self, days_old: int = 30) -> int:
+        """Return processing records older than the cutoff time optionally filtered by status."""
+        try:
+            query = self.db.query(ArtifactProcessing).filter(
+                ArtifactProcessing.created_at < cutoff_time
+            )
+
+            if status:
+                query = query.filter(ArtifactProcessing.status == status)
+
+            records = query.order_by(ArtifactProcessing.created_at.desc()).all()
+            return [ArtifactProcessingModel(**record.__dict__) for record in records]
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)